cmd/tail/main.go

```diff
@@ -26,11 +26,7 @@ func main() {
 	i := 0
 	s := strings.Split(string(data), "\n")
 	for j := len(s) - 1; j >= 0 && i < nvar; j-- {
```

Comment:
this would print the lines upside down? printed lines should be in the same order of the file